modules/core/src/main/scala/doobie/util/transactor.scala

```diff
@@ -50,12 +52,16 @@ object transactor  {
 
     /** Natural transformation that wraps a `ConnectionIO` program. */
     val wrap = λ[ConnectionIO ~> ConnectionIO] { ma =>
-      (before *> ma <* after) onError oops guarantee always
+      (before *> ma <* after)
+        .onError { case NonFatal(_) => oops }
+        .guarantee(always)
```

Comment:
My bad, it was a simple mistake in the original PR:
```scala
(before *> ma <* after) onError { 
  case NonFatal(_) => oops guarantee always 
}
```

Reply:
👍